stable/kommander/Chart.yaml

```diff
@@ -9,4 +9,4 @@ maintainers:
   - name: gracedo
   - name: dkoshkin
 name: kommander
-version: 0.39.2
+version: 0.39.3
```

Comment:
we can probably ignore changing these for the `kommander` chart - this is only used in 1.x